client/src/intro-exit/Introduction.jsx

```diff
@@ -1,11 +1,33 @@
-import React from "react";
+import React, { useEffect } from "react";
 import { Button } from "../components/Button";
+import { usePlayer } from "@empirica/player";
 
 export function Introduction({ next }) {
+  const player = usePlayer(); 
+
+  // async function calculateStart() {
+  //   const date = new Date(); 
+  //   const time = date.getTime(); 
+  //   console.log("intro start: " + time)
+
+  //   player.set("startT", time);
+  // }
+
+  // calculateStart();
+
+  useEffect(() => {
+    // localStorage.clear();
+    const date = new Date(); 
+    const time = date.getTime(); 
+    player.set("tStart", time);
```

Comment:
Let's move where we log player connection to the server side. I think for now we'll want to use the `onNewPlayer` callback. Down the road, if we have players connecting for multiple batches, we may want to do some checks that we're setting the right time for returning players,but for now let's keep it simple.

Reply:
You may end up needing a different date package on the server side, I'm not sure.